Rephrasing sentence to make it clearer after customer request (#806)
neo4j/docs-operations: modules/ROOT/pages/authentication-authorization/sso-integration.adoc

First, you configure Neo4j to use OpenID Connect as an authentication and authorization provider.

. Make sure security is turned on. 

> The `dbms.security.auth_enabled` setting is `true` by default.

Reviewer: ```suggestion
The default value for `dbms.security.auth_enabled` is `true`.
```

In this configuration, Neo4j receives a JWT from the identity provider containing claims representing the database username (e.g. email), and the Neo4j roles.

> . *Set a Display Name*

Reviewer: ```suggestion
. Set a display name.
```

----
This is displayed on a button on the login page of clients such as Neo4j Browser and Bloom, so that users can identify the provider they are using to login.

> . *Configure Discovery*

Reviewer: ```suggestion
. Configure discovery.
```

----
Manual settings always take priority over those retrieved from the discovery endpoint.

> . *Configure Audience*

Reviewer: ```suggestion
. Configure audience.
```

dbms.security.oidc.mysso.claims.client_id=myclientid
----

> . *Configure Claims*

Reviewer: ```suggestion
. Configure claims.
```

+
Provide the name of the claims that map to the database username and roles.
`username` is expected to be a string claim, and `roles` is expected to be a list of strings representing a set of roles or a single string representing a single role:
+
[source, properties]
----
dbms.security.oidc.mysso.claims.username=sub
dbms.security.oidc.mysso.claims.groups=roles
----

. *Optionally, map the groups in the OIDC groups claim to the Neo4j built-in and custom roles*
+
See xref:authentication-authorization/sso-integration.adoc#auth-sso-map-idp-roles[Map the Identity Provider Groups to the Neo4j Roles].

[[auth-sso-configure-provider-userinfo]]
=== OpenID Connect Fetching Claims from Provider

> In this configuration, Neo4j receives a token from the identity provider and uses that token to call back to the identity provider using its UserInfo endpoint to retrieve claims for the database username and Neo4j roles.

Reviewer: ```suggestion
In this configuration, Neo4j receives a token from the identity provider and uses that token to call back to the identity provider using its _UserInfo_ endpoint to retrieve claims for the database username and Neo4j roles.
```

> . *Configure as for JWT Claims*
> +
> Configure Neo4j for xref:authentication-authorization/sso-integration.adoc#auth-sso-configure-provider-jwt[OpenID Connect Using JWT Claims].

Reviewer: I think this looks a little repetitive, and can be simplified as a one-liner:

```suggestion
. Configure Neo4j for xref:authentication-authorization/sso-integration.adoc#auth-sso-configure-provider-jwt[OpenID Connect Using JWT Claims].
```

> . *Configure the claims to fetch from the UserInfo endpoint*

Reviewer: ```suggestion
. Configure the claims to fetch from the _UserInfo_ endpoint:
```
